manifesto.tex

```diff
@@ -107,37 +107,37 @@ \subsection{Operational Decentralisation}
 
 Secondly, as pointed out by Moxie recently \cite{TODO} major networks almost universally fail at facilitating decentralised usage of their service to their users. In order to gain the guarantees offered by the decentralised service, it is generally required to participate on the network directly by running the node software. This generally requires large amounts of time, a reliable, high-speed connection, plenty of storage and the installation of specialist software. Almost all real users are unable to do this and instead simply trust a centralised intermediary which relays information to and from the network.\footnote{Polkadot is actually far ahead of the pack on this, and can provide its service securely to users both in-browser or on mobile without the need for such an intermediary.} A notable example of this is the Infura infrastructure of Ethereum which is used by default in Metamask, the predominant way of accessing Ethereum-based applications.
 
-Thirdly, finally and most importantly for the present work is the practical means by which node operators fulfil their role. It must be understood that blockchains are inhererntly software systems: operating a node on the network involves securely running a piece of software on a machine able to correctly operate according to the network's protocol. We generally call this piece of software the ``blockchain client'' or the ``node software''. Node operators may a decision to run this software and must manually select the software to download and may even need to decide when and whether to apply updates to it. Most node operators are uninterested in arcane protocol discussions and defer to very basic social factors in their decision. This leads to a silent and hithertoo little-explored centralisation risk for the network. And, unlike the first two points which can arguably be solved purely through mathematics and technology, I posit that this is instead a more entrenched social issue and as such a harder problem to solve. To understand the centralisation risk, it is crucial to understand the factors by which node operators select \emph{which} software to use, and \emph{whether} they should apply some upgrade. If these factors lead to the possibility of a small, well-aligned group of people able to widely deploy changes to the protocol, then it can gravely undermine the protocol.
```

Comment:
GitHub seems to butcher this diff and doesn't show changed words; two words were changed here:

* `inhererntly` -> `inherently`
* `hithertoo` -> `hitherto`